lib/template/config/initializers/log.rb

```diff
@@ -1 +1,3 @@
-Pliny.default_context = {}
+Pliny.default_context = {
+  app: Config.app_name
```

Comment:
I think this is going to start logging `app=` when the config is not set, right?
Maybe only add the key "app" if the config is set?